package/yast2-country.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Nov 23 17:29:35 UTC 2022 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Update lang cache when selecting new lang
```

Comment:
"language", not "lang"
but more importantly:
You're describing the implementation of the fix.
It would be better to describe (also) the user benefit, the symptom of the solved bug.

Reply:
OK